CODING_GUIDELINES.md

```diff
@@ -0,0 +1,35 @@
+
+# Coding Guidelines
+
+This document contains the coding guidelines for the ASPIRE Python Project.
+
+## Coding Style
+
+In order to keep the code consistent, please read and follow [PEP 8 Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008)
+and [PEP 257 Docstring Conventions](https://www.python.org/dev/peps/pep-0257/) guidelines. [The webpage from RealPyhon](https://realpython.com/documenting-python-code/) is a good guide for documenting Python Code. 
```

Comment:
RealPyhon -> RealPython
The webpage -> This webpage ("from RealPython" shouldn't be in the hyperlink)

Reply:
fixed.